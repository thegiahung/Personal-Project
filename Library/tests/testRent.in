rent 100000 111111
add member Python
add member Java
rent 100000 111111
add book books.csv 111111
rent 100000 111111
relinquish 100000 111111
rent 100001 111111
exit
