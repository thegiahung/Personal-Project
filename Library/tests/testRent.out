user: No members in system.

user: Success.

user: Success.

user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Success.

user: Success.

user: Success.

user: Ending Library process.
